sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/OnHeapColumnVector.java

```diff
@@ -147,6 +147,18 @@ public void putBooleans(int rowId, int count, boolean value) {
     }
   }
 
+  @Override
+  public void putBooleans(int rowId, byte src) {
```

Comment:
👍 I think this is much easier to read :) we can always optimize it later if it turns out to be importance for performance (right now seems not so).

Reply:
Thanks, sounds good!